src/assets/scripts/InputController.js

```diff
@@ -467,17 +481,17 @@ export default class InputController {
 
             case KEY_CODES.LEFT_ARROW:
                 // shortKeys in use
-                if (window.gameController.game.option.get('controlMethod') === 'arrows') {
-                    this.$commandInput.val(`${currentCommandInputValue} \u2BA2`);
+                if (this._isArrowControlMethod()) {
+                    this.$commandInput.val(`${currentCommandInputValue}t l `);
```

Comment:
@n8rzz Not to be high maintenance, but could you add back the spaces before the command as well? The idea is to make them chainable, eg `[right arrow] [3] [5] [0] [down arrow] [5] [0] [numpad -] [1] [7] [0]`, and get `AAL 123 > 350 v 50 - 170`, so you never have to press space.